quickwit/quickwit-metastore/src/metastore/retrying_metastore/mod.rs

```diff
@@ -54,6 +56,12 @@ impl RetryingMetastore {
     }
 }
 
+impl fmt::Debug for RetryingMetastore {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
nitpick just to mark that I am actually reviewing stuff :D

```suggestion
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Reply:
I admire your commitment to removing lifetimes :D Copilot adds them. Do you want to make this part of our coding guidelines to reduce noise during code reviews?